scripts/kind.sh

```diff
@@ -14,14 +14,34 @@ source "$SCRIPTS_DIR/lib/logging.sh"
 setup_kind_cluster() {
     local __cluster_name=$1
     local __controller_namespace=$2
-    local kubeconfig_path="$ROOT_DIR/build/clusters/$__cluster_name/kubeconfig"
+    local kind_cluster_build_dir="$ROOT_DIR/build/clusters/$__cluster_name"
+    local kubeconfig_path="$kind_cluster_build_dir/kubeconfig"
+    local kubecontext="kind-$__cluster_name"
+    local kubecontext_path="$kind_cluster_build_dir/kubecontext"
 
     info_msg "Creating cluster with name \"$__cluster_name\""
     _create_kind_cluster "$__cluster_name" "$kubeconfig_path"
 
-    info_msg "Exporting KUBECONFIG=$kubeconfig_path"
-    export KUBECONFIG=$kubeconfig_path
+    info_msg "Creating kubecontext file $kubecontext_path ... "
+    cat <<EOF > "$kubecontext_path"
+export KUBECONFIG="$kubeconfig_path"
+export KUBECONTEXT="$kubecontext"
+kubectl config use-context "$kubecontext"
+kubectl config set-context --current --namespace=ack-system
 
+controller_pod() {
+  echo \$(kubectl get pods -ojsonpath='{.items[0].metadata.name}')
+}
```

Comment:
How/Where is this function called? I don't see any mentions of it in test-infra repo

Reply:
NVM i missed this
> Finally, the kubecontext file has a helper function calleld controller_pod that returns the Pod name for the controller under test.